### ISSUE TITLE
Error in fetchLatestVersion

### ISSUE THREAD

**Issue by autopulated** [lucagrulla/cw#32]
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x80 pc=0x142790f]

goroutine 21 [running]:
main.fetchLatestVersion.func1(0xc0001f2cc0)
	/Users/lucagrulla/Projects/cw/main.go:113 +0x3f
created by main.fetchLatestVersion
	/Users/lucagrulla/Projects/cw/main.go:110 +0x58
```

This might be because the network request to check for a new version is failing?

**Comment by lucagrulla** [comment]
Yes you're probably right. 

Sorry about that.
I'll do a quick patch over the next couple of days.

**Comment by lucagrulla** [comment]
Fixed in v2.1.3.
